app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+    has_many :votes
+    
+    validates :category, presence: true
+    validates :title, presence: true
+
+    def self.spotlight
+        @works = Work.all
+
+        top_voted = @works.max_by { |work| work.votes.count }
+        return top_voted
+    end
+
+    def self.top_ten(category)
+        media = Work.where(category: category)
+        top_media = media.sort_by { |work| work.votes.count }.reverse
```

Comment:
This works, but it's better to use `.order` and `.limit` to make postgres do the work of sorting and limiting the length of the response.   Postgres is much more efficient at it and it means postgres doesn't have to transmit the entire set of Work to Rails.